tests/components/aurora_abb_powerone/test_aurora_device.py

```diff
@@ -0,0 +1,48 @@
+"""Test the Aurora ABB PowerOne Solar PV config flow."""
+
+from aurorapy.client import AuroraSerialClient
+
+from homeassistant.components.aurora_abb_powerone.aurora_device import AuroraDevice
+from homeassistant.components.aurora_abb_powerone.const import (
+    ATTR_DEVICE_NAME,
+    ATTR_FIRMWARE,
+    ATTR_MODEL,
+    ATTR_SERIAL_NUMBER,
+    DEFAULT_INTEGRATION_TITLE,
+    DOMAIN,
+    MANUFACTURER,
+)
+from homeassistant.config_entries import ConfigEntry
+
+
+async def test_create_auroradevice(hass):
+    """Test creation of an aurora abb powerone device."""
+    client = AuroraSerialClient(7, "/dev/ttyUSB7", parity="N", timeout=1)
+    config = ConfigEntry(
+        version=1,
+        domain=DOMAIN,
+        title=DEFAULT_INTEGRATION_TITLE,
+        data={
+            ATTR_SERIAL_NUMBER: "65432",
+            ATTR_MODEL: "AAYYBB",
+            ATTR_DEVICE_NAME: "Feathers McGraw",
+            ATTR_FIRMWARE: "0.1.2.3",
+        },
+        source="dummysource",
+        entry_id="13579",
+    )
+    device = AuroraDevice(client, config.data)
```

Comment:
Setup the integration and assert that the device registry is populated instead of interacting directly

https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations

Reply:
This is already done by test_config_flow.py, There was already 100% coverage of aurora_device.py so I have deleted this file.